### DISCUSSION TITLE
my stars count is wrong

### DISCUSSION THREAD

**Discussion by quanzhuo** [community/community#19612] [Projects and Issues] [answered]
I starred 99 repos, but it displayed 294 stars 
![image](https://user-images.githubusercontent.com/8767119/176172237-14c1c854-5b21-47b0-829e-fa6ac9aa6f3f.png)

**Comment by melvincwng** [comment]
@quanzhuo, could it be that some of your repos have multiple stars? 

i.e. they were starred by other people as well?

**Reply by JhonnyLusonode** [reply]
Might be starred topics cause they also count. Check it out.
![image](https://user-images.githubusercontent.com/43578500/176183360-807ce9b5-100a-49d2-92ea-378b7b296a7e.png)

**Reply by melvincwng** [reply]
@JhonnyLusonode, good point Jhonny! +1 👍🏻 

Thanks for bringing this up!

@quanzhuo , FYI please note Jhonny's response too.

**Reply by JhonnyLusonode** [reply]
@melvincwng , 
Sure! i´m happy to help 👍

**Reply by quanzhuo** [reply]
@JhonnyLusonode , You are right. starred topic count too.  
I have 294 starred repos and topics in total. I removed some starred repos at about 17:00 (Beijing time), it's 4 hours ago. But the starred count didn't change when I open this issue (I have refresh the page for many times). When I reply this, it changed to 183. Maybe the github's starred count in cache is not changed in time.

![image](https://user-images.githubusercontent.com/8767119/176188887-f56d12bf-c0c0-48e0-8fed-2b3d49a76675.png)

**Reply by JhonnyLusonode** [reply]
@quanzhuo maybe it really is a bug.

Try reporting it to [Github Bug Report](https://support.github.com/contact/bug-report) over here.

Maybe they will give you the answer you need 😄

**Reply by quanzhuo** [reply]
> @quanzhuo maybe it really is a bug.
> 
> Try reporting it to [Github Bug Report](https://support.github.com/contact/bug-report) over here.
> 
> Maybe they will give you the answer you need 😄

Thank you, I will report this issue

**Comment by ghost** [comment, marked as answer]
I think it's a bug because this issue has happed to me too

**Comment by energister** [comment]
FYI: as far as I've noticed the counter only counts public repositories
